website/docs/reference/projects.md

```diff
@@ -3,131 +3,119 @@ id: projects
 title: Projects
 ---
 
-:::info feature availability
+## Overview
 
-All users get access to projects, but only pro and enterprise users can create, update, or delete them.
+Projects help you organize feature flags within Unleash. For example, you can use projects to group feature flags by development teams or different functional modules within your application.
 
-:::
-
-This document explains how Unleash uses projects, including how to create and maintain them.
-
-## The default project
-
-All users get access to the default project. You cannot delete this project. You can, however, rename it if you're using the pro or enterprise version of Unleash.
-
-## Understanding purpose of projects {#understanding-purpose-of-projects}
-
-Projects are a way to organize your feature flags within Unleash. Within a large organization, having multiple feature flags, staying on top of the feature flags might become a challenge. Every feature flag will be part of a project. Projects can be linked to a development team or to functional modules within the software.
+Each instance requires at least one project. New instances get a default project that all members can access. You cannot delete this project, but you can [rename](#update-a-project) or [archive it](#archive-a-project).
 
-A common pattern is to organize the feature flags according to key areas of the application, e.g. “Basic user process” and “Advanced user process”. This is illustrated below.
+You can configure which [environments](./environments.md) are available within a project. By default, all global environments are available. You can only enable and disable feature flags for the environments you've configured for the project.
 
-![A diagram with two boxes labeled "Basic user process" and "Advanced user process", respectively. The former contains features "New login" and "Winter theme enablement", the latter "New in-app purchase" and "Updated invoice repository".](/img/project_concept.png)
```

Comment:
Explained as an example in the Overview section, I think we can remove the screenshot as it doesn't really bring anything extra